tools/jdbc/src/main/java/org/duckdb/DuckDBResultSet.java

```diff
@@ -1194,7 +1194,8 @@ public <T> T getObject(int columnIndex, Class<T> type) throws SQLException {
 			}
 		} else if (type == Long.class) {
 			if (sqlType == DuckDBColumnType.BIGINT
-					|| sqlType == DuckDBColumnType.TIMESTAMP) {
+					|| sqlType == DuckDBColumnType.TIMESTAMP
```

Comment:
Might be worth pulling out these sets of comparisons? `isTimestampType`, `isLongType`, etc